network/wsNetwork_test.go

```diff
@@ -4516,7 +4492,7 @@ func TestMergePrimarySecondaryRelayAddressListsNoDedupExp(t *testing.T) {
 
 		network := supportedNetworkGen().Draw(t1, "network")
 		primaryDomainSuffix := strings.Replace(
-			`n-<network>.algorand0.network`, "<network>", network, -1)
+			`algorand-<network>.net`, "<network>", network, -1)
```

Comment:
How did this work before, couldn't it have drawn the non `n-<network>` suffix?

Reply:
The failures have been with the n- suffix, so I removed those as an option. I don't think that's the only problem, but want to run this several 1000 times first.